src/v2/view-builder/views/EnrollProfileView.js

```diff
@@ -107,13 +111,34 @@ export default BaseView.extend({
   },
   postRender() {
     BaseView.prototype.postRender.apply(this, arguments);
-
     const modelError = this.model.get('parseSchemaError');
 
     if (modelError) {
       this.model.trigger('error', this.model, {
         responseJSON: modelError,
       });
     }
-  }
+
+    // Prompt for password w/ SSR if enabled (credentials object in remediation)
+    const policy = this.getPasswordPolicySettings();
+    this.displayPasswordPolicy(policy);
```

Comment:
nit - since the `displayPasswordPolicy(policy)` depends on the `getPasswordPolicySettings()`, you can consider calling this in the `getPasswordPolicySettings()`. You can therefore skip returning the settings on line#137 and use it as a boolean to call the `displayPasswordPolicy()`

Reply:
Thanks again for the catch! Addressed